pkg/sfu/connectionquality/connectionstats.go

```diff
@@ -152,6 +199,14 @@ func (cs *ConnectionStats) updateStatsWorker() {
 			if cs.onStatsUpdate != nil {
 				cs.onStatsUpdate(cs, stat)
 			}
+			// Allow score transmission post as 1st cycle data can be partial for the interval
+			if counter == 1 {
```

Comment:
Is it possible to just skip onStatsUpdate when counter is < 2? and/or setting cs.score to 0.

Would be ideal not to keep another variable since it increases cognitive load.

Reply:
We don't want to skip onStatsUpdate, this is whats sent to telemetry for usage stats and scores.. For clients GetScore() is what is called, which reads cs.Score in parallel

Reply:
does this mean telemetry would receive a potentially inaccurate score then? it seem that they should be consistent.